app/dashboard/models.py

```diff
@@ -95,15 +93,11 @@ class Bounty(SuperModel):
     bounty_owner_email = models.CharField(max_length=255, blank=True)
     bounty_owner_github_username = models.CharField(max_length=255, blank=True)
     bounty_owner_name = models.CharField(max_length=255, blank=True)
-    fulfiller_address = models.CharField(max_length=50)
-    fulfiller_email = models.CharField(max_length=255, blank=True)
-    fulfiller_github_username = models.CharField(max_length=255, blank=True)
-    fulfiller_name = models.CharField(max_length=255, blank=True)
+    fulfillments = JSONField(default={})
```

Comment:
how does this JSONField differ from the foreign key to the `BountyFulfillment` table?

Reply:
Good question.  It doesn't really.  I felt that adding a new table was appropriate from a "database management" perspective, but I found the JSON field just easier to work with in general, because its easy in Javascript.

I think if we get into a situation where there are like hundreds of fulfillments, using the JSON field could be pretty slow and bulky, and there the extra table makes sense.  But for the case of just a few fulfillments, I think the JSON field is convenient and easy.

So I did it both ways.  I'm open to suggestions either way.

Reply:
i think we decided on a direction for this in slack, no?